### PR TITLE
Updated links

### PR DESCRIPTION
**Pull request by GeorgeColdham**
# Incorrect linking in the service guide top nav

[Ticket](https://jira.tools.tax.service.gov.uk/browse/CTDA-893)

### CODE REVIEW

File: config/tech-docs.yml [hunk 1]
```diff
@@ -3,15 +3,15 @@ host: docs.developer.service.hmrc.gov.uk
 
 # Header-related options
 show_govuk_logo: true
-service_name: CTC Traders API service guide
-service_link: https://developer.service.hmrc.gov.uk/guides/common-transit-convention-traders-service-guide
+service_name: HMRC Developer Hub
```
Comment (RobertLaverick) [inline config/tech-docs.yml:6]: Is this meant to be the Developer Hub?

File: config/tech-docs.yml [hunk 2]
```diff
@@ -3,15 +3,15 @@ host: docs.developer.service.hmrc.gov.uk
...
 # Links to show on right-hand-side of header
 header_links:
-  CTC Traders API Service Guide: /guides/CTC-traders-api-end-to-end-service-guide
-  Documentation: /api-documentation/docs/using-the-hub
-  Applications: /developer/applications
-  Support: /developer/support
+  CTC Traders API Service Guide: /
+  Documentation: https://developer.service.hmrc.gov.uk/api-documentation/docs/using-the-hub
+  Applications: https://developer.service.hmrc.gov.uk/developer/login
+  Support: https://developer.service.hmrc.gov.uk/developer/support
```
Comment (RobertLaverick) [inline config/tech-docs.yml:14]: Can you explain why these are absolute references?

File: config/tech-docs.yml (hunk 1 above)
Comment (GeorgeColdham) [reply config/tech-docs.yml:6]: Yep, from clarification this morning

File: config/tech-docs.yml (hunk 2 above)
Comment (GeorgeColdham) [reply config/tech-docs.yml]: Sorted

**Review by RobertLaverick — APPROVED** [review]

**Pull request merged**